fbpic/fields/numba_methods.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2016, FBPIC contributors
```

Comment:
Here are the new numba functions for the CPU.

Except for the loop over iz and ir, they are indentical to the corresponding cuda functions. 
Maybe we could build a numba "inline function" to avoid code duplication.
@MKirchen, @Escapado  : I know less than you do on this. Do you have any suggestions?